app/uk/gov/hmrc/gform/controllers/AuthenticatedRequestActions.scala

```diff
@@ -114,7 +114,7 @@ class AuthenticatedRequestActions(
                          lang,
                          request.uri,
                          getAffinityGroup,
-                         ggAuthorised(request)(authUserWhitelist(_)))
+                         ggAuthorised(request)(AuthSuccessful(_).pure[Future]))
```

Comment:
See how all calls ggAuthorised now call with the parameter authGivenRetrievals as AuthSuccessful(_).pure[Future], so that parameter is no longer required.  I suspect m most of that function collapses and can be removed